relay-general/src/store/normalize.rs

```diff
@@ -745,12 +746,37 @@ pub struct LightNormalizationConfig<'a> {
     pub transaction_name_config: TransactionNameConfig<'a>,
     pub is_renormalize: bool,
     pub device_class_synthesis_config: bool,
-    pub scrub_span_descriptions: bool,
+    pub enrich_spans: bool,
     pub light_normalize_spans: bool,
+    pub max_tag_value_length: usize, // TODO: move span related fields into separate config.
     pub span_description_rules: Option<&'a Vec<SpanDescriptionRule>>,
     pub geoip_lookup: Option<&'a GeoIpLookup>,
 }
 
+impl Default for LightNormalizationConfig<'_> {
+    fn default() -> Self {
+        Self {
+            client_ip: Default::default(),
+            user_agent: Default::default(),
+            received_at: Default::default(),
+            max_secs_in_past: Default::default(),
+            max_secs_in_future: Default::default(),
+            max_name_and_unit_len: Default::default(),
+            measurements_config: Default::default(),
+            breakdowns_config: Default::default(),
+            normalize_user_agent: Default::default(),
+            transaction_name_config: Default::default(),
+            is_renormalize: Default::default(),
+            device_class_synthesis_config: Default::default(),
+            enrich_spans: Default::default(),
+            light_normalize_spans: Default::default(),
+            max_tag_value_length: usize::MAX,
+            span_description_rules: Default::default(),
+            geoip_lookup: Default::default(),
```

Comment:
Would it make sense to write something like 
```suggestion
            max_tag_value_length: usize::MAX,
            ..Default::default()

```

Reply:
`..Default::default()` takes the values from the containing type's default, not the field's default, so unfortunately not.